client/app/components/dashboards/EditParameterMappingsDialog.jsx

```diff
@@ -45,7 +39,8 @@ class EditParameterMappingsDialog extends React.Component {
     widget
       .save()
       .then(() => {
-        this.props.close();
+        this.props.onChange();
```

Comment:
Maybe rename this prop to something like `onAccept` (and use this name in all modals)? WDYT?

Reply:
👍🏻onConfirm?